en/editor/publish/publish-in-command-line.md

```diff
@@ -33,9 +33,9 @@ Currently, when using the command line to build, except for the required build o
 
 - `--build`: Specify the parameters to be used when building the project.
 
-  If no parameters are specified after `--build`, then the parameters used in the **Build** panel, such as platforms, templates, and so on, will be used as default parameters. If additional parameter settings are specified, the default parameters will be overwritten with the specified parameters. The available parameters are:
+  - When no parameters are specified after `--build`, then the parameters used in the **Build** panel, such as platforms, templates, and so on, will be used as default parameters. When additional parameter settings are specified, the default parameters will be overwritten with the specified parameters. The available parameters are:
```

Comment:
When no parameters are specified
-->
When parameters are not specified

I apologize for the additional change. When I re-read this out loud I need to change it.

Reply:
OK，I will fix it.